<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Files to convert</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="removeFile">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="upFile">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="downFile">
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget">
         <property name="minimumSize">
          <size>
           <width>399</width>
           <height>230</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Converted files</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="minimumSize">
          <size>
           <width>399</width>
           <height>220</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="removeRippedFile">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="upRippedFile">
         <property name="text">
          <string>Up</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="downRippedFile">
         <property name="text">
          <string>Down</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Toolbar</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="12" column="2" colspan="2">
        <widget class="QPushButton" name="cleanAll">
         <property name="text">
          <string>Clean All</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLCDNumber" name="durationView">
         <property name="minimumSize">
          <size>
           <width>93</width>
           <height>54</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>93</width>
           <height>54</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="4">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Commandline commands output</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="2">
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QPushButton" name="manualRename">
         <property name="text">
          <string>Manual Rename</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QCheckBox" name="autoRename">
         <property name="text">
          <string>Rename files automatically</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="startButton">
         <property name="maximumSize">
          <size>
           <width>109</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Convert Files</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="stopButton">
         <property name="maximumSize">
          <size>
           <width>109</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Stop CDRipper </string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="durationButton">
         <property name="text">
          <string>Calculate duration</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>54</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>94</width>
           <height>54</height>
          </size>
         </property>
         <property name="text">
          <string>Total duration :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="albumName">
         <property name="maximumSize">
          <size>
           <width>157</width>
           <height>21</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Album name :</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAddfile"/>
    <addaction name="actionAddripped"/>
    <addaction name="actionCDTracks"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionNewProject"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Convert files</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop CDRipper</string>
   </property>
  </action>
  <action name="actionAdd_file">
   <property name="text">
    <string>Add file(s)</string>
   </property>
  </action>
  <action name="actionAdd_ripped_file">
   <property name="text">
    <string>Add ripped file(s)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCDTracks">
   <property name="text">
    <string>Rip and Add CD Drive tracks</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionAddfile">
   <property name="text">
    <string>Add files</string>
   </property>
  </action>
  <action name="actionAddripped">
   <property name="text">
    <string>Add ripped files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
